src/test/java/org/mule/extension/ftp/PostActionGroupTestCase.java

```diff
@@ -40,4 +63,194 @@ public void deleteAndMove() {
   public void deleteAndRename() {
     new PostActionGroup(true, null, "thisone.txt", true).validateSelf();
   }
+
+
+  @Test(expected = FileAlreadyExistsException.class)
+  @Description("verifies that move file with overwrite flag set to false and exceptiopn will be thrown")
+  public void testMoveWithoutOverwriteWhenFileExists() {
+    FileAttributes fileAttributes = mock(FileAttributes.class);
+    when(fileAttributes.getName()).thenReturn("");
+    FileConnectorConfig fileConnectorConfig = mock(FileConnectorConfig.class);
+
+    ConcreteFileSystem fileSystem = new ConcreteFileSystem("");
+
+    new PostActionGroupChild("someDir", null, false, false).apply(fileSystem, fileAttributes, fileConnectorConfig);
```

Comment:
Tests should include assertions. Add at least one assertion to this test case

Reply:
This is to test exception got thrown correctly. So I think no assertion here.
This kind of test case also exists in file common.

Reply:
Sorry, I wasn't clear, I mean that you should put something like this... 
`fail("Expected exception was not thrown");`
but no worry. I approve the ticket. It is not important